examples/petsc/bps.c

```diff
@@ -141,6 +168,16 @@ int main(int argc, char **argv) {
     ierr = DMPlexCreateFromFile(PETSC_COMM_WORLD, filename, PETSC_TRUE, &dm);
     CHKERRQ(ierr);
   } else {
+    if (userlnodes) {
+      // Find a nicely composite number of elements no less than lnodes
```

Comment:
`lnodes` -> `localnodes`?

Reply:
We have `localnodes`. This `userlnodes` (which refers to user local nodes for short) was just a flag to determine whether the user had set at CL option for nodes or not (otherwise the mesh is created using the global `-cells` parameter)

Reply:
'twas a poorly worded phone comment. There is no `lnodes` in the code and the comment isn't correct anyway. I'm happy with any wording that makes the comment accurate (or delete the comment).

Reply:
Ah, gotcha!